multiple-instances/rust/src/main.rs

```diff
@@ -52,6 +54,53 @@ async fn get_sdk(
     Ok(sdk)
 }
 
+/// On first run (if you don't already have a node), leave `mnemonic` as None. A new mnemonic is
+/// generated and printed to console.
+///
+/// On subsequent runs, or if you already have a node, set the `mnemonic`.
+///
+/// The device key `client-key.pem` and certificate `client.crt` are read from the current directory.
+///
+/// If you have an invite code instead of a partner certificate, use [get_sdk_with_invite_code].
+async fn get_sdk_with_partner_certificate(
+    breez_sdk_api_key: &str,
+    working_dir: &str,
+    mnemonic: Option<&str>,
+) -> Result<Arc<BreezServices>> {
+    let mnemonic_obj = match mnemonic {
+        None => {
+            let mnemonic = Mnemonic::generate_in(Language::English, 12)?;
+            println!("Generated mnemonic: {mnemonic}");
+            mnemonic
+        }
+        Some(mnemonic_str) => Mnemonic::from_str(mnemonic_str)?,
+    };
+
+    let seed = mnemonic_obj.to_seed("");
+
+    let mut config = BreezServices::default_config(
+        EnvironmentType::Production,
+        breez_sdk_api_key.into(),
+        breez_sdk_core::NodeConfig::Greenlight {
+            config: GreenlightNodeConfig {
+                partner_credentials: Some(GreenlightCredentials {
+                    device_key: std::fs::read("client-key.pem")?,
+                    device_cert: std::fs::read("client.crt")?,
+                }),
+                invite_code: None,
+            },
+        },
+    );
```

Comment:
Suggestion to make it a little clearer these are read from disk.
```suggestion
    let client_key = std::fs::read("client-key.pem")?;
    let client_cert = std::fs::read("client.crt")?;
    let mut config = BreezServices::default_config(
        EnvironmentType::Production,
        breez_sdk_api_key.into(),
        breez_sdk_core::NodeConfig::Greenlight {
            config: GreenlightNodeConfig {
                partner_credentials: Some(GreenlightCredentials {
                    device_key: client_key,
                    device_cert: client_cert,
                }),
                invite_code: None,
            },
        },
    );
```